rust/src/logger.rs

```diff
@@ -0,0 +1,39 @@
+//! Logging module
```

Comment:
Compatibility: this way, you can use crates that use the debug! etc. macros without changing a single line of code
Testing: you can test your parsers outside Suricata (debug! will send to stderr) or inside (to Suricata's logging facility)